coalib/parsing/Globbing.py

```diff
@@ -252,14 +252,20 @@ def _absolute_flat_glob(pattern):
     return
 
 
-def _iter_relative_dirs(dirname):
+def _iter_relative_dirs(dirname, ignored_globs=None):
     """
     Recursively iterates subdirectories of all levels from dirname
 
+    :param ignored_globs: List of globs to ignore when matching files
     :param dirname: Directory name
     :return:        Iterator that yields files and directory from the given dir
                     and all it's (recursive) subdirectories
     """
+    if ignored_globs != None:
+        for ignored_dir in ignored_globs:
+            if dirname.endswith(ignored_dir):
```

Comment:
I don't think this will work because

```python
ignored_dir == 'env/**'
```

or

```python
ignored_dir == 'vendor/*/node_modules/**'
```

etc.